src/components/Cart/Cart.tsx

```diff
@@ -0,0 +1,108 @@
+import { useSelector, useDispatch } from "react-redux";
+
+import { RootState } from "store/store";
+
+import { cartActions } from "store/cartSlice";
+
+import styled from "styled-components";
+
+import CartItem from "./CartItem";
+
+import { CartStyleProps } from "interfaces/Cart";
+
+import {
+  respondFrom,
+  breakpoints,
+  errorText,
+  dimensions,
+  fonts,
+  colors,
+} from "styles";
+
+const CartWrapper = styled.div<Pick<CartStyleProps, "variant">>`
+  display: flex;
+  flex-direction: column;
+  gap: ${dimensions.spacing.md}px;
+  ${(props) =>
+    props.variant === "menu" &&
+    `position: absolute; top: 42px; right:0; padding: 50px; width:500px;  background-color: ${colors.white}; border-bottom-left-radius:17px; border-bottom-right-radius:17px;`}
```

Comment:
let's format this into separate lines for each css property so it is more legible

Reply:
Done